device/src/u3v/async_read.rs

```diff
@@ -203,6 +203,9 @@ impl AsyncTransfer {
 impl Drop for AsyncTransfer {
     fn drop(&mut self) {
         unsafe {
+            drop(Box::from_raw(
+                self.transfer().user_data.cast::<AtomicBool>(),
```

Comment:
I wonder if this comment should be updated, or if it was originally referencing this `Drop` impl where it was forgotten.

https://github.com/cameleon-rs/cameleon/blob/c0cebd17d4d595e69c23a0332215cdffbf840bc2/device/src/u3v/async_read.rs#L142-L144

Reply:
Yeah, my gut feeling is that the comment is correct but the implementation wasn't, so this should be OK to keep.